### PR TITLE
Creating a theme guide

### PR DESCRIPTION
**Pull request by eballantine**
Based on the [guidance in notion](https://www.notion.so/easol/Creating-custom-themes-theme-developer-709f4ae092de4a369a6c0e5f5204017f)

Adapted for a public audience.

### CODE REVIEW

**Review by paulina-a — APPROVED** [review]
Thanks @eballantine for kicking this off - feels like a good base which we can later iterate on as the process evolves. Added a few notes but looks good to me.

**Comment by paulina-a** [comment]
One thing I forgot to mention in the review is that this will also need the schema adding at the top

e.g.
```
---
layout: default
title: Author
parent: Objects
---
```

File: docs/guides/getting_started/creating_a_theme.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+---
+
+## Step 3: Develop your theme
+
+A few things to keep in mind while you develop your theme:
+
+From the moment a theme is chosen for a site, a copy of all the theme code is added to that site. If you make changes to your theme code, the newer version does not replace the theme code in existing sites. This has a few effects worth thinking about: 
+- As you develop and test your theme, all other theme files (e.g. theme css, js) needs to be updated manually in the Easol company admin under **Site > Theme**.
+- Updates to the block library will only affect the blocks available for adding to pages. Existing blocks already added to site pages will not be updated.
```
[the remark this answers is not in the document]
Comment (eballantine) [reply docs/guides/getting_started/creating_a_theme.md]: Agreed. In Nina's version 'block library' linked to additional resources. I've added an empty page for the 'blocks' documentation and linked to that, and updated the wording for clarity. Will merge this to keep things moving, but further feedback Paulina, Ian, Kyle, v welcome!

File: docs/guides/getting_started/creating_a_theme.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+---
+
+## Step 3: Develop your theme
+
+A few things to keep in mind while you develop your theme:
+
+From the moment a theme is chosen for a site, a copy of all the theme code is added to that site. If you make changes to your theme code, the newer version does not replace the theme code in existing sites. This has a few effects worth thinking about: 
+- As you develop and test your theme, all other theme files (e.g. theme css, js) need to be updated manually in the Easol company admin under **Site > Theme**.
+- Updates to [blocks]({% link docs/guides/getting_started/creating_a_theme.md %}) will not update any blocks which have already been added to site pages. However, changes will appear in the block library, which is presented when editing pages. This can lead to having multiple versions of the same block present in the same site.
```
Comment (ianmooney) [inline docs/guides/getting_started/creating_a_theme.md:39]: Is the `blocks` link intended to link somewhere else?

**Review by ianmooney — COMMENTED** [review]
Looks good to me! Nice to see these docs coming to life 🙌

**Pull request merged**